packages/react-cosmos-ui/src/plugins/FixtureTree/FixtureTree/FixtureButton.tsx

```diff
@@ -22,8 +25,16 @@ export function FixtureButton({
   selectedRef,
   onSelect,
 }: Props) {
+  const { pluginContext } = usePlugContext<FixtureTreeSpec>();
+  const root = pluginContext.getMethodsOf<RootSpec>('root');
+
+  function handleSelect(fixtureId: FixtureId) {
+    onSelect(fixtureId);
+    if (root.getFloatingPanes()) root.closeFixtureList();
+  }
```

Comment:
Maybe create a FixtureTreeContext with the following data:
- expansion
- setExpansion
- selectFixture

As well as a FixtureTreeScrollContext with

- containerRef
- selectedRef